### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by microo8** [gorgonia/golgi#1]
Has anyone started to implement this?
It would be great to have a keras alternative to on gorgonia.

**Comment by chewxy** [comment]
Yeah it is... I made a mistake sometime in November - I coded most of Golgi
in the vendor subpackage, and then proceeded to delete it.

So it's taking some effort to get it back up. I'll pull up a copy tonight

*Xuanyi Chew*
*@chewxy on Twitter*
*Read:* my blog <http://blog.chewxy.com>
*+61403928398*

**Comment by ghost** [comment]
Oops...
I also am interested.
The docs are very well written btw in the repos - hats off

**Comment by RameshRavone** [comment]
+1

**Comment by zhenghaoz** [comment]
+1

**Comment by steele232** [comment]
I'm excited to see this and try this out. I see some code committed 5 days ago but I haven't looked at it quite yet. Just the filenames look exciting.

**Comment by steele232** [comment]
I see that some of them are just stubs, but what I see looks good. I'm hoping this will be something that is super user friendly on top of gorgonia, kind of like Keras can be for Tensorflow.

**Comment by chewxy** [comment]
@steele232 that's the plan. I wrote the stubs because I had expected me to finish porting over my personal library to this over the weekend. Two things prevented this: 

1. Life got in the way (furniture building takes more hours than expected)
2. I added a new term to the calculus that defines the language here but when I put it thru Coq I couldn't prove correctness so I had to go back to the drawing board for a bit. 

Having said that, the Coq bits don't really affect much. I'll try to get the other layers in.
